### DISCUSSION TITLE
How to link manual tests to autotests?

### DISCUSSION THREAD

**Discussion by deimiana** [allure-framework/allure2#1743] [TestOps]
Hello! I use Allure TMS, nUnitAllure adapter for c# and Teamcity. I have some manual tests for automation in Allure TMS also I have this test automated. I have integration between TMS and teamcity and reports are succesfully upload into launches. My autotests were created as new tests. But I can't understand how to link my manual tests to autotests from reports. Help me please. I tryed to google and read documentation but it didn't make more clear.

**Comment by cheshi-mantu** [comment]
@deimiana 
Hi there
this is described here: https://docs.qameta.io/allure-testops/ecosystem/intellij-plugin/#link-tests-in-ide-to-tests-cases-in-allure-testops-via-allureid
a) you need to link automated test via AllureID
b) you need to upload test results for the said test case to Allure TestOps, this would finalise the linking
